pylint_odoo/checkers/modules_odoo.py

```diff
@@ -484,6 +485,16 @@ def _get_duplicate_xml_fields(self, fields):
             all_fields.setdefault(field_xml, []).append(field)
         # Remove all keys which not duplicated
         for key, items in all_fields.items():
+            parents = [e.getparent() for e in items]
+            same_parent = [
+                item for item, count in collections.Counter(parents).items()
+                if count > 1]
```

Comment:
@moylop260 I'm not a fan of multiline generators, but it's not so bad in this case because the logical separation is also good visually.

I dunno if McCabe is the only stat that should be relevant here though. I would create an example of something with a lower complexity that is significantly less readable than the explicit cousin, but my `pylint` doesn't seem to have `--max-complexity` option. Are you using a beta version or something? 

```
pylint: error: no such option: --max-complexity
```